common/predictive-text/testing/dummy-model/index.js

```diff
@@ -0,0 +1 @@
+../../build/index.js
```

Comment:
I think I know why I didn't push this code earlier — it relies on a symbolic link to the TS compiled file. Symlinks don't play well with Windows, in my experience, so this might not work universally :/